packages/client/components/DashboardAvatars/DashboardAvatars.tsx

```diff
@@ -64,93 +54,107 @@ const StyledButton = styled(PlainButton)({
   WebkitTapHighlightColor: 'transparent',
   '&:hover': {
     cursor: 'pointer'
-  }
+  },
+  marginTop: sizeToHeightBump[ElementWidth.DASHBOARD_AVATAR] * -1 // coupling with AvatarList styles
```

Comment:
+1 if it needs to be a negative margin, I'd prefer to put a `-sizeToHeightBump[ElementWidth.DASHBOARD_AVATAR]` as the multiplication implies a calculation is required. 

I haven't dug into the calculations in this PR review yet, but in general, I'd love to minimise the number of calculations in this PR. They may all be necessary, but there's a lot of calculations for marginTop, minHeight, minWidth. If we can use the box model as much as possible so that we don't need to define the exact heights * margins, we'll save ourselves plenty of pain down the line.

Reply:
I've removed calculation usage.

Regarding the number of calculations, I had a hard time finding a better solution.
I'm more than happy to hear some ideas on how to handle this differently!